returnn/datasets/hdf.py

```diff
@@ -1403,6 +1405,7 @@ def dump_from_dataset(self, dataset, epoch=1, start_seq=0, end_seq=float("inf"),
 
     # Again iterate through dataset, and set the data
     print("Write data...", file=log.v3)
+    dataset.epoch = None  # force re-init (inconsistency in MetaDataset)
```

Comment:
Why a workaround for another bug?
Why not fix that other bug ("inconsistency in MetaDataset")?